types/react-draft-wysiwyg/index.d.ts

```diff
@@ -1,11 +1,12 @@
-// Type definitions for react-draft-wysiwyg 1.12
+// Type definitions for react-draft-wysiwyg 1.13
 // Project: https://github.com/jpuri/react-draft-wysiwyg#readme
 // Definitions by: imechZhangLY <https://github.com/imechZhangLY>
 //                 brunoMaurice <https://github.com/brunoMaurice>
 //                 ldanet <https://github.com/ldanet>
 //                 Munif Tanjim <https://github.com/MunifTanjim>
+//                 Nathan Zeplowitz <https://github.com/n-zeplo>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.9
+// TypeScript Version: 3.6
```

Comment:
People don't usually *actually* want to update this version as it requires the user have TS 3.6 to use this package. @imechZhangLY

Reply:
Reverted back as per your suggestion. Thanks for the heads up. Please, find the commit below.